website/versioned_docs/version-0.20/faq-developers.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
This shouldn't be added by hand. The docusaurus is:
1. add content to `./docs` 
2. generate the versions when changes are ready (which will generate this file)

I think this is the more complete text? so I think this should be move / replace the corresponding in `./docs`